src/com/lftechnology/java/training/sanish/application/component/EmployeeHelper.java

```diff
@@ -0,0 +1,416 @@
+ *
+ * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+ */
+public class EmployeeHelper {
+    private static final Logger LOGGER = Logger.getLogger(EmployeeHelper.class.getName());
+
+    private EmployeeHelper() {
+    }
+
+    /**
+     * Change user name of user
+     *
+     * @param inputScanner {@link Scanner}
+     * @param user         {@link User}
+     * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+     */
+    public static void changeUserName(Scanner inputScanner, User user) {
+        String message = "\n" + Constants.USER_NAME_LABEL + user.getUserName() + "\n>>" + Constants.CHANGE_USER_NAME_CONFORM_MSG;
+        LOGGER.log(Level.INFO, message);
+        String edit = UserInput.getString(inputScanner);
+        if (edit.equals(Constants.YES_OPTION) || edit.equals(Constants.FULL_YES_OPTION)) {
+            int updateRow = 0;
+            String userName;
+            String SetStatement;
+            String whereCondition = "userId=?";
+            UserDao userDao = new UserDao();
+            while (updateRow == 0) {
+                message = "\n >> " + Constants.TYPE_USER_NAME_LABEL;
+                LOGGER.log(Level.INFO, message);
+                userName = UserInput.getString(inputScanner);
+                SetStatement = "userName=?, modifiedAt=?";
+                DateFormat dateFormat = new SimpleDateFormat(Constants.DB_DATE_FORMAT);
+                Date date = new Date();
+                updateRow = userDao.update(whereCondition, SetStatement, userName, dateFormat.format(date), user.getUserId());
+                if (updateRow == 0) {
+                    message = "\n" + Constants.ERROR_MSG_LABEL + Constants.FAIL_UPDATE_USER_NAME_MSG + Constants.TRY_AGAIN_MSG;
+                    LOGGER.log(Level.INFO, message);
+                    edit = UserInput.getString(inputScanner);
+                    if (!edit.equals(Constants.YES_OPTION) && !edit.equals(Constants.FULL_YES_OPTION)) {
+                        break;
+                    }
+                }
+            }
+        }
+    }
+
+    /**
+     * Change email of user
+     *
+     * @param inputScanner {@link Scanner}
+     * @param user         {@link User}
+     * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+     */
+    public static void changeUserEmail(Scanner inputScanner, User user) {
+        String message = "\n" + Constants.USER_EMAIL_LABEL + user.getEmail() + "\n>>" + Constants.CHANGE_EMAIL_CONFORM_MSG;
+        LOGGER.log(Level.INFO, message);
+        String edit = UserInput.getString(inputScanner);
+        if (edit.equals(Constants.YES_OPTION) || edit.equals(Constants.FULL_YES_OPTION)) {
+            int updateRow = 0;
+            String email;
+            String SetStatement;
+            String whereCondition = "userId=?";
+            UserDao userDao = new UserDao();
+            while (updateRow == 0) {
+                message = "\n >>" + Constants.TYPE_EMAIL_LABEL;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "\n >>" 15 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)